### PR TITLE
[PLAYER-1771] Add hiding/showing toolbar after change video mode and …

### PR DESCRIPTION
**Pull request by atopilskii**
…replace default toolbar with support toolbar to avoid shifting toolbar after changing video mode.

### CODE REVIEW

File: VRSampleApp/app/src/main/java/com/ooyala/sample/MainActivity.java [hunk 1]
```diff
@@ -71,4 +80,22 @@ public boolean onOptionsItemSelected(MenuItem item) {
...
 		return false;
 	}
+
+  @Override
+  public void update(Observable o, Object arg) {
+    super.update(o, arg);
+    changeToolbarVisibilityInFullscreenMode(arg);
+  }
+
+  private void changeToolbarVisibilityInFullscreenMode(Object arg) {
```
Comment (IvanSakharovskii) [inline VRSampleApp/app/src/main/java/com/ooyala/sample/MainActivity.java:92]: Is this problem appear only for our sample app? Maybe we need to move it into skin

File: VRSampleApp/app/src/main/res/layout/toolbar_default.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v7.widget.Toolbar xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (IvanSakharovskii) [inline VRSampleApp/app/src/main/res/layout/toolbar_default.xml]: code style. And for toolbar we can use `Toolbar.setTitle`

File: VRSampleApp/app/src/main/java/com/ooyala/sample/MainActivity.java (hunk 1 above)
Comment (atopilskii) [reply VRSampleApp/app/src/main/java/com/ooyala/sample/MainActivity.java:92]: They don't use toolbar, so there is nothing to fix.

File: VRSampleApp/app/src/main/java/com/ooyala/sample/MainActivity.java [hunk 3]
```diff
@@ -14,15 +17,21 @@
...
 	private static final String TAG = "VRSampleApp";
+  Toolbar toolbar;
 
   @Override
   protected void onCreate(Bundle savedInstanceState) {
     super.onCreate(savedInstanceState);
 
     setContentView(R.layout.activity_main);
+    toolbar = (Toolbar) findViewById(R.id.toolbar);
+    setSupportActionBar(toolbar);
```
Comment (IvanSakharovskii) [inline VRSampleApp/app/src/main/java/com/ooyala/sample/MainActivity.java]: check that toolbar is not null

**Review by IvanSakharovskii — APPROVED** [review]

**Pull request merged**